core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -163,14 +168,21 @@ public KafkaSchemaRegistry(SchemaRegistryConfig config,
    * of say the last, is arbitrary.
    */
   // TODO: once RestConfig.PORT_CONFIG is deprecated, remove the port parameter.
-  static int getPortForIdentity(int port, List<String> configuredListeners) {
+  static AbstractMap.SimpleEntry<String, Integer> getSchemeAndPortForIdentity(
```

Comment:
If there's some library with a less awkward Pair or Tuple implementation that you want to use, it should be safe to pull in here as this is an app and not a library. Or just define a simple class for this -- it's internal so just a struct-like class with two public fields would work fine and provides clearer naming than `getKey` and `getValue`